CanvasPlusPlayground/PeopleManager.swift

```diff
@@ -23,23 +23,27 @@ class PeopleManager {
     func fetchPeople() async {
         guard let courseID else { return }
 
-        let _: [Enrollment]? = try? await CanvasService.shared.defaultAndFetch(
+        let enrollments: [Enrollment]? = try? await CanvasService.shared.loadAndSync(
             .getPeople(courseId: courseID),
+            descriptor: .init(sortBy: [
+                SortDescriptor(\.user?.name, order: .forward)
+            ]),
             onCacheReceive: { (cached: [Enrollment]?) in
                 guard let cached else { return }
 
                 let users = cached.compactMap { $0.user }
                         .filter { user in !self.users.contains(where: { $0.id == user.id }) }
 
-                self.users.append(contentsOf: Set(users))
-            },
-            onNewBatch: { batch in
```

Comment:
why are we removing `onNewBatch`?

Reply:
it's not being used correctly in this case (it's adding duplicate elements), was thinking we can modify it later